files/build_templates/sonic_debian_extension.j2

```diff
@@ -626,6 +626,15 @@ sudo cp $IMAGE_CONFIGS/hostname/hostname-config.service $FILESYSTEM_ROOT_USR_LIB
 echo "hostname-config.service" | sudo tee -a $GENERATED_SERVICE_FILE
 sudo cp $IMAGE_CONFIGS/hostname/hostname-config.sh $FILESYSTEM_ROOT/usr/bin/
 
+{% if enable_local_users_passwords_reset == "y" %}
```

Comment:
do you need check sonic_asic_platform == "mellanox" as well?

Reply:
No, why? it is not related to Mellanox or ASIC.. it is a general feature

Reply:
ok I see, but I only saw [local_users_passwords_reset.py](https://github.com/sonic-net/sonic-buildimage/pull/18874/files#diff-3f39d7f91c00e60cdb70d4529ffd5ca8f913e3705047a8b20334870a26f5e587) for mellanox.

Reply:
The implementation is for Mellanox, each vendor should add their implementation in case they want to use it same as we did in mellanox